include/camp/resource/omp_target.hpp

```diff
@@ -189,6 +189,10 @@ namespace resources
         }
         return ret;
       }
+
+      int get_device() const { return dev; }
```

Comment:
Can you document these functions?

Reply:
+1 on documentation :D

Reply:
If you give me descriptions, I can help with the PR. The more docs the better :D

Reply:
I can document them. They are equivalent to the get_device and get_stream methods on the Cuda and Hip resources.